.travis.yml

```diff
@@ -1,36 +1,38 @@
 language: c++
+branches:
+  only:
+  - master
 cache:
   ccache: true
   directories:
-   - .cache/
-   - node_modules/
-   - $V8WORKER2_OUT_PATH
+   - deno2/js/node_modules/
+   - $DEPOT_TOOLS_PATH
+   - $BUILD_PATH
 env:
   global:
-    - V8WORKER2_OUT_PATH=$HOME/v8worker2_out
-    - PROTOBUF_ROOT=$HOME/protobuf
+    - BUILD_PATH=$HOME/out/Default
+    - DEPOT_TOOLS_PATH=$HOME/depot_tools
 before_install: |
-  # protobuf
-  export PATH=$PROTOBUF_ROOT/bin:$PATH
-  if ! [ -x $PROTOBUF_ROOT/bin/protoc ]; then
-    rm -rf $PROTOBUF_ROOT
-    mkdir -p $PROTOBUF_ROOT
-    pushd $PROTOBUF_ROOT
-    wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
-    unzip protoc-3.1.0-linux-x86_64.zip
-    popd
+  if ! [ -x $DEPOT_TOOLS_PATH/gclient ]; then
+    rm -rf $DEPOT_TOOLS_PATH
+    git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH
   fi
 install:
- - mkdir -p $GOPATH/src/github.com/ry
- - ln -s `pwd` $GOPATH/src/github.com/ry/deno
+ - unset CC CXX
+ - export PATH=$PATH:$DEPOT_TOOLS_PATH
+ - cd deno2
+ - gclient sync --no-history
+ # ccache needs this
+ - export CCACHE_CPP2=yes
+ - export CCACHE_SLOPPINESS=time_macros
+ - export PATH=`pwd`/third_party/llvm-build/Release+Asserts/bin:$PATH
+ - (cd js; yarn)
+ - rm -f $BUILD_PATH/args.gn # Don't use cached values.
+ - gn gen $BUILD_PATH --args='is_debug=false cc_wrapper="ccache" use_custom_libcxx = false use_sysroot = false'
```

Comment:
spaces around the equals sign (`use_sysroot = false`). That works?

Reply:
Fixed.